Delcam.ProductInterface.PowerMILL.Test/CollectionTests/PMToolpathsCollectionTest.cs

```diff
@@ -66,6 +67,17 @@ public void NoActiveItemTest()
             Assert.AreEqual(activeToolpath, null);
         }
 
+        [Test]
+        public void OrderByExplorerTest()
+        {
+            _powerMILL.LoadProject(TestFiles.ToolpathsOrderProject);
+            var startingOrder = _powerMILL.ActiveProject.Toolpaths.Select(x => x.Name);
```

Comment:
Might be worth adding an Assert on the starting order. "Nothing up my sleeve" and all that :-)

Reply:
Requested changes made.